gerrychain/partition/partition.py

```diff
@@ -51,6 +55,11 @@ def _first_time(self, graph, assignment, updaters):
         self.flows = None
         self.edge_flows = None
 
+    def validate_assignment(self):
+        node_names = set(list(self.graph.nodes()))
```

Comment:
I think we can shorten this to:
```suggestion
        node_names = set(self.graph.nodes)
```